feder/letters/logs/views.py

```diff
@@ -39,6 +44,61 @@ def monitoring(self):
         return get_object_or_404(Monitoring, pk=self.kwargs['monitoring_pk'])
 
 
+class EmailLogMonitoringCsvView(ListMonitoringMixin, ListView):
+    permission_required = 'monitorings.view_log'
+
+    select_related = ['case', 'case__institution']
+
+    @cached_property
+    def monitoring(self):
+        return get_object_or_404(Monitoring, pk=self.kwargs['monitoring_pk'])
+
+    def get(self, *args, **kwargs):
+        response = self._get_csv_response()
+        self._write_rows(response, self.get_queryset())
+        return response
+
+    @staticmethod
+    def _get_base_model_field_names(queryset):
+        opts = queryset.model._meta
+        return [field.name for field in opts.fields if field.related_model is None]
+
+    def _get_csv_response(self):
+        csv_response = HttpResponse(content_type='text/csv')
+        current_time = datetime.datetime.now(tzlocal())
```

Comment:
Od tego są mechanizmy w Django, które biorą strefę z konfiguracji Django, a nie konfiguracji serwera. Zobacz https://docs.djangoproject.com/en/1.11/topics/i18n/timezones/#naive-and-aware-datetime-objects .

Reply:
poprawione, dzięki za info!